project/en-US/compensations/compensations.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd" [
+        <!ENTITY % BOOK_ENTITIES SYSTEM "../project_documentation.ent">
+        %BOOK_ENTITIES;
+        ]>
+<chapter>
+    <title>Compensations Framework</title>
+    <section>
```

Comment:
Maybe reference the blogs.

"The Compensations Framework is still in development so the documentation is evolving. For now you can see the notes below plus take a look at our blog series: <link/>"